File: common/src/test/java/dev/number6/generate/SentimentScoreGenerator.java
package dev.number6.generate;

import com.amazonaws.services.comprehend.model.SentimentScore;
import uk.org.fyodor.generators.Generator;

public class SentimentScoreGenerator implements Generator<SentimentScore> {

    @Override
    public SentimentScore next() {
        return new SentimentScore()
                .withMixed(CommonRDG.sentimentScoreFloat())
                .withNegative(CommonRDG.sentimentScoreFloat())
                .withPositive(CommonRDG.sentimentScoreFloat())
                .withNeutral(CommonRDG.sentimentScoreFloat());
    }
}

File: common/src/test/kotlin/dev/number6/generate/SentimentScoreGenerator.kt
package dev.number6.generate

import com.amazonaws.services.comprehend.model.SentimentScore
import uk.org.fyodor.generators.Generator

class SentimentScoreGenerator : Generator<SentimentScore> {
    override fun next(): SentimentScore {
        return SentimentScore()
                .withMixed(CommonRDG.sentimentScoreFloat())
                .withNegative(CommonRDG.sentimentScoreFloat())
                .withPositive(CommonRDG.sentimentScoreFloat())
                .withNeutral(CommonRDG.sentimentScoreFloat())
    }
}